build_tools/docker/swiftshader/Dockerfile

```diff
@@ -17,7 +17,7 @@ WORKDIR /install-swiftshader
 
 RUN apt-get update && apt-get install -y git
 
-ARG SWIFTSHADER_COMMIT=6287c18b1d249152563f0cb2d5cb0c6d0eb9e3d6
+ARG SWIFTSHADER_COMMIT=84f5eeb6dd9b225f465f93737fa76aad7de355cf
```

Comment:
Please also rebuild the docker images and update the commit used in the RBE image (which has to be separate because it uses Ubuntu 16.04). See https://github.com/google/iree/blob/main/build_tools/docker/README.md I might need to give you some permissions in our cloud project